src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
I don't mean the funcitonality of the handler, that I get ok. I mean where would it rely on being in the default group for handler selection?